hypershift-operator/controllers/manifests/controlplaneoperator/manifests.go

```diff
@@ -48,6 +48,14 @@ func OperatorClusterRoleBinding(controlPlaneOperatorNamespace string) *rbacv1.Cl
 	}
 }
 
+func OperatorMonitoringClusterRoleBinding(controlPlaneOperatorNamespace string) *rbacv1.ClusterRoleBinding {
```

Comment:
TODO: Add a note that this needs replaced with a namespace scoped role binding if possible; according to the monitoring team it might be possible